src/workflows/pyproject.toml

```diff
@@ -0,0 +1,81 @@
+[tool.poetry]
+name = "airqo_etl_utils"
+version = "1.0.0"
+description = "AirQo Airflow custom python package"
+authors = ["AirQo <data@airqo.net>"]
+license = "MIT"
+keywords = ["python", "airflow", "AirQo"]
+include = ["**/*.json"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+apache-airflow = "*"
+apache-airflow-providers-slack = "*"
+confluent_avro = "*"
+google-cloud-bigquery = "*"
+google-cloud-storage = "*"
```

Comment:
google-cloud-storage = ">=2.0.0,<3.0.0"

Reply:
On it, Thanks for the feedback.

Reply:
Just ensure that you have the correct versions for the main packages just to be on the safe side at least for now. i.e those that are not dependencies of other packages.

You can list those packages with `pip list --not-required`